composer.json

```diff
@@ -231,7 +231,8 @@
         "symfony/runtime": "^5.4.21 || ^6.0",
         "symfony/web-profiler-bundle": "^5.4.21 || ^6.0",
         "symplify/monorepo-builder": "^11.0",
-        "vimeo/psalm": "5.9.*"
+        "vimeo/psalm": "5.12.*",
+        "weirdan/doctrine-psalm-plugin": "^2.8"
```

Comment:
In version `5.12`, Psalm started reporting false-positivies in one of the entities. Installing this plugin solved the problem without any modification in our config.